app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
And I thought about separating the schema again: So when we want to add a property to a entity, do we need to change the schema then? Or do we just keep the old schema since it can have additional properties anyway?

If we need to change the schema everytime we change an entity I still think we should keep the entities and the schema together, both the project and the version number, because otherwise it is too complicated to extend an entity, so you first need to extend the schema, then release it, then change the federation-library, release it and then add support for the property in diaspora. Then it's **much** easier to just add the property here in the federation library, both in the entity and the schema and then set the schema version to the next release-version of this gem, and release both together.

If we only change the schema from time to time and can add properties to entities without changing the schema, I'm fine with separate it to it's own project and own versioning.